.github/workflows/integration.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       - id: project
         uses: equinix-labs/metal-project-action@v0.13.0
         env:
-          METAL_AUTH_TOKEN: ${{ secrets.METAL_AUTH_TOKEN }}
+          METAL_AUTH_TOKEN: ${{ env.TF_VAR_auth_token }}
```

Comment:
Just want to call out that the integration test workflow has never passed; it's possible that was just because of the incorrect working-directory in the old file, but we don't know.  So: if you change this back to `secrets.METAL_AUTH_TOKEN` and the workflow still fails we shouldn't necessarily block this PR for that failure, but we should come back and fix the workflow quickly.

Reply:
It still fails with either `secrets.METAL_AUTH_TOKEN` or `${{ env.TF_VAR_auth_token }}`

This is the error I get: `You must provide an auth token in `with.userToken` must be supplied`

It looks like I am missing userToken var somewhere per the [docs](https://github.com/equinix-labs/metal-project-action#input)